### PR TITLE
chore: update and improve README.md

### LINKED ISSUE #838: Fix Readme for New Contributors
Trying to set up a local instance of ubiquibot for the first time and the setup isnt clear even for me so i cant imagine new contributors without context.

Redo the entire readme. Just delete and create a new file. Make sure to start from a brand new clone and go through every step of setup and document correctly. 

Make sure to clean up .env.example as well

from https://t.me/UbiquityDAO/29891/31238

---

I learned that the webhook secret is not optional. somewhere I read that it was. a contributor told me to just set it to something random on both the .env and the github app so this should be made clear.

also the error messages are really not descriptive and do not allow for debugging.

### PR DESCRIPTION
**Pull request by molecula451**
Resolves #838

Quality Assurance: https://github.com/molecula451/ubiquibot/blob/update-readme/README.md

### CODE REVIEW

**Review by wannacfuture — COMMENTED** [review]
Looks Good to me. 👍

This will help new contributors who having issues running the bot.
@ubiquity/software-development Please take a look

**Review by rndquu — DISMISSED** [review]

**Review by wannacfuture — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -115,7 +179,25 @@ SUPABASE_KEY="XXX"
...
+yarn supabase status
+```
+
+![supabase](https://github.com/ubiquity/ubiquibot/assets/41552663/e8709b8f-e7c3-49e0-876c-c15dde22c6d2)
+
+
+
+## Logs
+
+2. Create a new organization at [Mezmo](https://app.mezmo.com/). Add `LOGDNA_INGESTION_KEY` to the `.env` file:
```
Comment (cuddle-dev) [inline README.md]: Looks like it has been deprecated. We don't use logdna for the real-time logger. moved them to the supabase.

Comment (cuddle-dev) [reply README.md]: @molecula451

File: README.md [hunk 2]
```diff
@@ -10,17 +25,19 @@ Ubiquity DAO's GitHub Bot to automate DevPool management.
...
+
+yarn build --watch (to locally auto compile your changes)
 yarn start:watch
 ```
 
 ## Environment Variables
 
-- Copy `.env.example` to `.env`
-- Update `.env` with the following fields:
+- Copy `.env.example` to `.env` (do not rname .env.example, just make a copy)
```
Comment (whilefoo) [inline README.md]:
```suggestion
- Copy `.env.example` to `.env` (do not rename .env.example, just make a copy)
```

File: README.md [hunk 3]
```diff
@@ -98,13 +159,16 @@ To test the bot, you can:
...
 
-`promotion-comment` is a message that is appended to the payment permit comment.
+`promotionComment` is a message that is appended to the payment permit comment.
 
 ## How to run locally
+### Supase Database
+
+### Option 1
 
 1. Create a new project at [Supabase](https://supabase.com/). Add `Project URL` and `API Key` to the `.env` file:
```
Comment (whilefoo) [inline README.md:215]: Maybe you can clarify where to get URL and API key. 
Something like `Go to Project Settings and click on the API section, get the Project URL and API key for service_role/secret.`

**Review by whilefoo — CHANGES_REQUESTED** [review]

**Comment by molecula451** [comment]
everyone check QA https://github.com/molecula451/ubiquibot/blob/update-readme/README.md

File: README.md (hunk 1 above)
Comment (molecula451) [reply README.md]: done purged everything about mezmo @0xcodercrane

**Comment by molecula451** [comment]
it looks like pavlovcik wants to also integrate the supabase migration thing https://github.com/ubiquity/ubiquibot/issues/838#issuecomment-1734546986 but! the supabase readme it's complete and i think a dev should be aware of this module at the repo and how to handle the inner stuff of supabase, e.g migrations what do you guys think? @rndquu @0xcodercrane @wannacfuture @whilefoo

**Comment by 0x4007** [comment]
> everyone check QA https://github.com/molecula451/ubiquibot/blob/update-readme/README.md

I skimmed this. It looks really substantial! Good work! I do not have time to test the set up right now but I expect that the other team members should try from A to Z to set up a new bot and follow that tutorial.

**Comment by wannacfuture** [comment]
> > everyone check QA https://github.com/molecula451/ubiquibot/blob/update-readme/README.md
> 
> I skimmed this. It looks really substantial! Good work! I do not have time to test the set up right now but I expect that the other team members should try from A to Z to set up a new bot and follow that tutorial.

I have tried this readme(setting up the bot) and looks good to me. 👍

**Review by 0x4007 — APPROVED** [review]
I approve based on @wannacfuture's review. However it would be appreciated if we could get at least one more before merging.

**Comment by molecula451** [comment]
yeah i agree @rndquu @0xcodercrane

**Review by rndquu — APPROVED** [review]

**Comment by molecula451** [comment]
@0xcodercrane crane let's get an eagle eye on this one to help pavlovcik get it merge

**Review by wannacfuture — APPROVED** [review]

**Comment by 0x4007** [comment]
@whilefoo when you approve this will automatically merge. Thanks everybody for your reviews!

**Comment by molecula451** [comment]
@whilefoo your review it's important for everyone here

**Review by whilefoo — APPROVED** [review]

**Pull request merged**